spark/build.gradle.kts

```diff
@@ -38,6 +38,12 @@ android {
             "-opt-in=com.adevinta.spark.ExperimentalSparkApi",
         )
     }
+    sourceSets {
+        named("test") {
+            java.srcDirs("src/test/kotlin")
```

Comment:
nit: I don't think we should re-add this sourceSet.

Reply:
yep I forgot to delte it